.github/workflows/go-docker.yaml

```diff
@@ -0,0 +1,73 @@
+# see the bottom of the file for more information¹
+# the parent workflow is responsible for setting up the call events²
+on:
+  workflow_call:
+
+# it wasn't needed to set up anything for GoLang because 'ubuntu-latest' has all dependencies
+jobs:
+  # first stage
+  code_check-style_golangci_lint:
+    name: 'code-check > style:golangci-lint'
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: 'rios0rios0/pipelines/github/golang/stages/10-code-check/golangci-lint@feat/#19'
+    if: "!startsWith(github.ref, 'refs/tags/')"
+
+
+  # second stage
+  security-sast_horusec:
+    name: 'security > sast:horusec'
```

Comment:
There's no way to designate the "stage" (this concept doesn't exist here in GitHub Actions). So, I can't say what the jobs are inside the "stage" named X, to have the things organized. Like, inside the stage for "code-check" I have 3 jobs, each one with different tools and checks. All those 3, are dependent on the next STAGE and not the next JOB. Got it? Like we usually have in GitLab.

Reply:
That's why the names are like that. In the format "STAGE > JOB (TOOL)". To have a minimal of category and organization.